src/components/FileDetail/FileDetail.js

```diff
@@ -32,131 +40,155 @@ const FileDetail = ({fileNode, history, match}) => {
+                        onClick={e => {
+                          deleteFile({variables: {kfId: fileNode.kfId}});
+                          history.goBack();
+                        }}
+                        content="DELETE"
+                      />
+                    </Button.Group>
+                  </Header>
+                </Grid.Column>
+              </Grid.Row>
+              <Grid.Row stretched>
+                <Grid.Column mobile={8} tablet={8} computer={4}>
+                  <Statistic size="mini">
+                    <Statistic.Label style={{textAlign: 'left'}}>
+                      Status
+                    </Statistic.Label>
+                    <Statistic.Value style={{textAlign: 'left'}}>
+                      <Badge state={sortedVersions[0].node.state} />
+                    </Statistic.Value>
+                  </Statistic>
+                </Grid.Column>
+                <Grid.Column mobile={8} tablet={8} computer={4}>
+                  <Statistic size="mini">
+                    <Statistic.Label style={{textAlign: 'left'}}>
+                      Document Type
+                    </Statistic.Label>
+                    <Statistic.Value style={{textAlign: 'left'}}>
+                      <Label icon basic size="tiny">
+                        <Icon
+                          name={`${fileTypeDetail[fileNode.fileType].icon ||
+                            'question'}`}
                         />
-                      </div>
-                      <p className="FileInfo--Text">
-                        {fileTypeDetail[fileNode.fileType].title}
-                      </p>
-                    </div>
-                  </div>
-                  <div className="cell-6 sm:cell-3">
-                    <p className="FileInfo--Title">Last Updated:</p>
-                    {latestDate ? (
-                      <div className="flex">
-                        <p className="FileInfo--Text">
-                          <TimeAgo
-                            date={latestDate}
-                            live={false}
-                            className="mr-4"
-                          />
-                          by
-                        </p>
-                        <Avatar
-                          className="inline-block ml-4 mt-4"
-                          size={20}
-                          imgUrl={
+                        {' ' + fileTypeDetail[fileNode.fileType].title}
+                      </Label>
+                    </Statistic.Value>
+                  </Statistic>
+                </Grid.Column>
+                <Grid.Column mobile={8} tablet={8} computer={4}>
+                  <Statistic size="mini">
+                    <Statistic.Label style={{textAlign: 'left'}}>
+                      Last Updated
+                    </Statistic.Label>
+                    <Statistic.Value style={{textAlign: 'left'}}>
+                      <Label image size="tiny" style={{textWrap: 'no'}}>
```

Comment:
if this is how we are going to use the avatar and timeago lockup it should be moved to its own component for easier and consistent re-use